src/app/tabs/test-tab.js

```diff
@@ -151,7 +152,9 @@ module.exports = class TestTab extends ViewPlugin {
     return new Promise((resolve, reject) => {
       let runningTest = {}
       runningTest[path] = { content }
-      remixTests.runTestSources(runningTest, () => {}, () => {}, (error, result) => {
+      let currentCompilerUrl = this.baseurl + '/' + this.compileTab.getCurrentVersion()
+      let usingWorker = this.compileTab.compilerContainer.browserSupportWorker()
```

Comment:
is that one `browserSupportWorker` already defines?

Reply:
Yes, we added that here: https://github.com/ethereum/remix-ide/pull/2352#discussion_r326173883